src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -393,9 +394,11 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
       )}
       {isForSale && (
         <OwnerInformationContainer>
-          <ThemedText.BodySmall color="textSecondary" lineHeight="20px">
-            Seller:
-          </ThemedText.BodySmall>
+          {asset.owner.address && (
```

Comment:
might need to cast as a bool, still seeing it here `nfts/asset/0xded56b58644373020393ad35b9eb5841296b886b/1`

Reply:
ohh this was because its an erc1155